<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cargarPushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CARGAR ARCHIVO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UrlLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TxtPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>821</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lexico</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="LexplainTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="semanticoTextEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>560</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSemantico">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Semantico</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
